CHANGELOG.md

```diff
@@ -76,6 +77,9 @@ All notable changes to this project will be documented in this file.
         * Removed array-size parameter from apply_tdr_offsets() and write_source_waveform_broadcast_u32() methods - [#1070](https://github.com/ni/nimi-python/issues/1070)
         * Renamed `write_source_waveform_broadcast_u32()` to `write_source_waveform_broadcast()`
         * `get_pin_results_pin_information()` - returns namedtuple `PinInfo(pin_indexes, site_numbers, channel_indexes)`
+* ### NI Switch Executive
+    * #### Changed
+        * Version updated to 1.1.4 to match other drivers
```

Comment:
"driver" is ambiguous.
We talk about "driver runtime" to refer to what NI calls "driver". This is not what you mean.

You should say "other released nimi-python modules" - this way we don't include nitclk and nidigital.

Reply:
Done